### ISSUE TITLE
Accessibility: WCAG 2.0 2.4.7 violation for buttons

### ISSUE THREAD

**Issue by akhramov** [rust-lang/beta.rust-lang.org#474]
# Design Feature Request
## Summary
pt. 2.4.7 of WCAG 2.0 indicates that 
> Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible

However, focus on buttons is not visible. I suggest to add visible border outline, the same we have for `:hover`

## Motivation
This feature will help users with motor disabilities understand the site flow better.

## Drawbacks
I see no drawbacks

## Rationale and alternatives
It's the simplest and the most convenient thing.

## Maintenance
This is a one-time change.

## Unresolved Questions
None

**Comment by ashleygwilliams** [comment]
Thank you so much for filing! I see that you already filed a PR, I'll review it soon! Thanks again!
